templates/js/dashboard-tab/infra/azure.bicep

```diff
@@ -1,27 +1,24 @@
 @maxLength(20)
 @minLength(4)
 param resourceBaseName string
-param storageSku string
 
-param storageName string = resourceBaseName
-param location string = resourceGroup().location
+param staticWebAppName string = resourceBaseName
 
-// Azure Storage that hosts your static web site
-resource storage 'Microsoft.Storage/storageAccounts@2021-06-01' = {
-  kind: 'StorageV2'
-  location: location
-  name: storageName
-  properties: {
-    supportsHttpsTrafficOnly: true
-  }
+// Azure Static Web Apps that hosts your static web site
+resource swa 'Microsoft.Web/staticSites@2022-09-01' = {
+  name: staticWebAppName
+  // SWA do not need location setting
+  location: 'centralus'
   sku: {
-    name: storageSku
+    name: 'Free'
+    tier: 'Free'
```

Comment:
Can the tier be a parameter just as storageSku?

Reply:
Fixed